site/docs/master/httpproxy.md

```diff
@@ -252,6 +252,53 @@ The TLS **Minimum Protocol Version** a vhost should negotiate can be specified b
 - 1.2
 - 1.1 (Default)
 
+##### Fallback Certificate
+
+Contour provides virtual host based routing, so that any TLS request is routed to the appropriate service based on both the server name requested by the TLS client and the HOST header in the HTTP request. 
+
+As HOST Header is encrypted during TLS handshake, it can’t be used for virtual host based routing unless client sends HTTPS request specifying hostname using the SNI or the request is first decrypted using a default TLS certificate.
+Some TLS clients do not sent the server name, so Envoy does not know how to select the right certificate, therefore a fallback certificate is needed for these clients. 
```

Comment:
```suggestion
Some legacy TLS clients do not send the server name, so Envoy does not know how to select the right certificate. A fallback certificate is needed for these clients.
```

Reply:
Why does it have to be "legacy" clients?

Reply:
Because SNI has been standardized for at least a decade. SNI is our preferred path because we are virtual hosting and don't support the usual workarounds (certificates per address, etc).

Reply:
Right but I don't know why we need to categorize it as such. Built 20 years ago or last week as "cloud-native", if the client doesn't support SNI, then this is the path they need to follow. 

I'll add "legacy" so that we can get this closed for the release next week.